src/app/messages/SimpleMessageProvider.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useState } from 'react';
+import { IntlShape, RawIntlProvider } from 'react-intl';
+import createIntl from './createIntl';
+
+// tslint:disable-next-line:variable-name
+const SimpleMessageProvider = (props: { children?: React.ReactNode }) => {
+  const [intl, setIntl] = useState<IntlShape | null>(null);
+
+  useEffect(() => {
+    const setUpIntl = async() => {
+      const languageWithCode = window!.navigator.language || 'en-US';
+      const language = new (Intl as any).Locale(languageWithCode).language;
```

Comment:
The type for this is available in Typescript 4.5, so I figured it was desirable to wait to upgrade rather than pull in a package like `@formatjs/intl-locale`. But I can still pull that in, or regex it out instead.

Reply:
if its on `window.Intl` you may just need to change `src/typings/global.d.ts` to expose it

we have some awkward typing going on around the browser globals to avoid them being defined when they shouldn't in ssr